services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/SystemStoreCheckStats.java

```diff
@@ -1,21 +1,26 @@
 package com.linkedin.venice.controller.stats;
 
 import com.linkedin.venice.stats.AbstractVeniceStats;
+import com.linkedin.venice.stats.Gauge;
 import io.tehuti.metrics.MetricsRepository;
 import io.tehuti.metrics.Sensor;
-import io.tehuti.metrics.stats.Count;
 
 
 public class SystemStoreCheckStats extends AbstractVeniceStats {
-  private final Sensor badSystemStoreCount;
+  private final Sensor badMetaSystemStoreCount;
+  private final Sensor badPushStatusSystemStoreCount;
 
   public SystemStoreCheckStats(MetricsRepository metricsRepository, String name) {
     super(metricsRepository, name);
+    badMetaSystemStoreCount = registerSensorIfAbsent("bad_meta_system_store_count", new Gauge());
```

Comment:
This is not the right way to use `Gauge` metric and maybe we should just remove the default constructor of `Gauge`.
We need to pass a metric value provider when constructing the Gauge metric and you can see the existing usage.

Reply:
Oh right... I think I remember that.